### PR TITLE
Fix prompts not being printed by disabling stdout buffering

### PR DESCRIPTION
**Pull request by jirutka**
When built against musl libc, prompts like "Master key:", "Username:" etc. are not printed to console because of stdout buffering.

### CODE REVIEW

**Comment by jesperpedersen** [comment]
Would you be interested in porting over the improvements that was done in pgmoneta to pgexporter 

https://github.com/pgmoneta/pgmoneta/blob/main/src/cli.c
https://github.com/pgmoneta/pgmoneta/blob/main/src/admin.c

?

Or going with your current patch...

**Comment by jirutka** [comment]
I don’t understand, which changes from pgmoneta to port and how is it related to this issue?

**Comment by jesperpedersen** [comment]
The same issue should be fixed in pgmoneta...

But, we can go with your patch too

**Review by resyfer — APPROVED** [review]

**Comment by resyfer** [comment]
TL;DR LGTM.

A `NULL` value is same as `_IONBF` acc to [man page](https://man7.org/linux/man-pages/man3/setbuf.3.html), so all good. `pgexporter` does not print too much to `stdout` for this to impact performance imo, so this should be fine instead of `fflush`.

**Comment by jesperpedersen** [comment]
Merged.

Thanks for your contribution !
